clients/client-python/README.md

```diff
@@ -16,16 +16,16 @@
     ```bash
     pip install -e '.[dev]'
     ```
-   
-2. Run tests
-    ```bash
-    cd gravitino
-    ./gradlew :clients:client-python:test
-    ```
 
-3. Run integration tests
+2. Run integration tests
     ```bash
     cd gravitino
     ./gradlew compileDistribution -x test
-    ./gradlew :clients:client-python:integrationTest
+    ./gradlew :clients:client-python:test
     ```
+
+3. Test mode
+
+    + Test Principle: Every Python ITs class base on the `IntegrationTestEnv`, `IntegrationTestEnv` will automatically start and stop Gravitino server to support ITs, But when you run multiple ITs class at same time, The first test class that finishes running will shut down the Gravitino server, which will cause other test classes to fail if they can't connect to the Gravitino server.
+    + Run test in the IDE: Through `IntegrationTestEnv` class automatically start and stop Gravitino server to support ITs.
+    + Run test in the GitHub Action or Gradle command `:client:client-python:test`: Gradle automatically start and stop Gravitino server, and set `START_EXTERNAL_GRAVITINO` environment variable.
```

Comment:
What do you want to describe here. I think you explained lots of things but without actions. You'd better list some instructions on how to use it, if it is not user-related, you don't have to explain a lot.

Reply:
This part of description need be polished; Maybe you can try GTP for it. Besides, I think for the Python user, we need a guide to tell them how to use this library.